3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/EnrichmentManager.scala

```diff
@@ -231,7 +235,25 @@ object EnrichmentManager {
       event.page_urlquery = components.query.orNull
       event.page_urlfragment = components.fragment.orNull
     }
-
+    
+    // For parsing ValidationString (value) to String type
+    def tryParse(s: scalaz.Validation[String,String]):String = {
+      s match {
+        case Success(pass) => {
+          pass
+        }
+        case Failure(failure) => {
+          failure
+        }
+      }
+    }
+    // For Currency Conversion Enrichment
+    val trTotal = tryParse(CU.stringToDoublelike("tr_total", "tr_tt" ))
+    val trTax = tryParse(CU.stringToDoublelike("tr_tax", "tr_tx" ))
+    val trShipping =  tryParse(CU.stringToDoublelike("tr_shipping", "tr_sh" ))
+    val tiPrice = tryParse(CU.stringToDoublelike("ti_price", "ti_pr" ))
+    val tiCurrency = tryParse(CU.stringToDoublelike("ti_currency", "ti_cu" ))
```

Comment:
Currency is a string like GBP, it's not something that should be parsed to a Double.